arch/arm/soc/atmel_sam0/samd20/soc_pinmap.h

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Note that this has moved to pinmux.c.  Consider doing the same in a follow up CL.